l10n/en/products/vpn/platforms/desktop.ftl

```diff
@@ -30,7 +30,7 @@ vpn-desktop-servers-headline = Connect to servers all over the world
 
 # Variables:
 #   $servers (number) - number of available servers
-vpn-desktop-servers-copy = Browse from Brazil. Game from Japan. Stream from Mexico. { -brand-name-mozilla-vpn } lets you change your phone or computer’s location to one of { $servers }.
+vpn-desktop-servers-copy = Browse from Brazil. Game from Japan. Stream from Mexico. { -brand-name-mozilla-vpn } lets you change your phone or computer’s location to one of { $servers } servers.
```

Comment:
Do you need to update this string, in case localizers have already started?

Reply:
Example from [German](https://pontoon.mozilla.org/de/mozillaorg/en/products/vpn/platforms/desktop.ftl/?string=221152)

Reply:
Ah! I didn't realized this was already in Pontoon, I thought it was still on hold. I'll add the fixed string as new.